src/EnergyPlus/ZoneEquipmentManager.cc

```diff
@@ -4041,16 +4041,12 @@ namespace EnergyPlus::ZoneEquipmentManager {
                         // Zone has no infiltration objects
                         MassConservation(ZoneNum).InfiltrationMassFlowRate = 0.0;
                     }
-
+                }
+                //
+                if (ZoneMassBalanceFlag(ZoneNum)) {
                     MassConservation(ZoneNum).InMassFlowRate = state.dataZoneEquip->ZoneEquipConfig(ZoneNum).TotInletAirMassFlowRate;
                     MassConservation(ZoneNum).ExhMassFlowRate = state.dataZoneEquip->ZoneEquipConfig(ZoneNum).TotExhaustAirMassFlowRate;
-                    ZoneMixingNetAirMassFlowRate = MassConservation(ZoneNum).MixingMassFlowRate - MassConservation(ZoneNum).MixingSourceMassFlowRate;
                 }
-                //
-
-                TotSupplyAirMassFlowRate = state.dataZoneEquip->ZoneEquipConfig(ZoneNum).TotInletAirMassFlowRate -
-                                           (state.dataZoneEquip->ZoneEquipConfig(ZoneNum).TotExhaustAirMassFlowRate - state.dataZoneEquip->ZoneEquipConfig(ZoneNum).ZoneExh) -
-                                           state.dataZoneEquip->ZoneEquipConfig(ZoneNum).PlenumMassFlow;
```

Comment:
Removed unused code section.

Reply:
Good catch. Yours truly left this hanging here some years back during the multilple airloops serving the same zone rewrite.